src/App.js

```diff
@@ -66,7 +68,7 @@ function App() {
   const fetchRecipies = async () => {
     try {
       const response = await fetch(
-        "https://api.spoonacular.com/recipes/complexSearch?apiKey=9621d08e65f74b1abbcf1c8549fbde66"
+        `https://api.spoonacular.com/recipes/complexSearch?apiKey=${REACT_APP_API_KEY}`
       );
 
       const response_data = await response.json();
```

Comment:
seems that you're not using this fetch?

Reply:
yes, because of the API points, I replaced it with local array of the objects which is noramlly returned by the API.